sources/packages/backend/apps/api/src/services/education-program/education-program.service.ts

```diff
@@ -268,16 +268,14 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
     // For getting total raw count before pagination.
     const sqlQuery = paginatedProgramQuery.getSql();
 
-    if (paginationOptions.pageLimit) {
+    if (paginationOptions.pageLimit > 0) {
```

Comment:
if  `pageLimit` min is set to 0 then we can use the old condition itself right?

Reply:
-1 is for all records. i have added in the code comment

Reply:
I saw the comment but made this comment with respect to whether should use Min as -1 for the pagelimit.